generators/app/tasks/fileCreators.js

```diff
@@ -30,7 +30,10 @@ const getPackageJsonAttributes = (projectName, projectVersion, repoUrl) => ({
     build: 'node ./scripts/build.js',
     // eslint-disable-next-line no-extra-parens
     ...(this.customized && { deploy: 'node ./scripts/deploy.js' }),
-    test: generateRSScript('test', '--env=jsdom'),
+    'ts-check': 'tsc',
```

Comment:
```suggestion
    'tsc': 'tsc',
```
Maybe I'd rename the name just for clarity. Not sure 🙃